lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -13,6 +13,7 @@
   // so maybe we add the border-radius to all media-* classes?
   & .media-video {
     border-radius: 4px;
+    padding-bottom: 60.4%;
```

Comment:
Yeah, was thinking there might be a better approach. Will add comment for now to get this shipped.